### ISSUE TITLE
HParams Scatter Plot Matrix View including metrics on x-axis

### ISSUE THREAD

**Issue by sumanthratna** [tensorflow/tensorboard#3063]
### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version 3b8c68cd4c94fc96d7f998dfcbc1b953b6de6c0e

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=6, micro=8, releaselevel='final', serial=0)
INFO: os.name: posix
INFO: os.uname(): posix.uname_result(sysname='Darwin', nodename='SumanthRatnasAir', release='19.3.0', version='Darwin Kernel Version 19.3.0: Sun Dec  8 22:27:29 PST 2019; root:xnu-6153.80.8.0.1~15/RELEASE_X86_64', machine='x86_64')
INFO: sys.getwindowsversion(): N/A

--- check: package_management
INFO: has conda-meta: False
INFO: $VIRTUAL_ENV: '/Users/suman/.local/share/virtualenvs/mlencrypt-research-PwGoqeJm'

--- check: installed_packages
INFO: installed: tensorboard==2.0.2
INFO: installed: tensorflow==2.0.0
INFO: installed: tensorflow-estimator==2.0.1

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.0.2'

--- check: tensorflow_python_version
INFO: tensorflow.__version__: '2.0.0'
INFO: tensorflow.__git_version__: 'v2.0.0-rc2-26-g64c3d382ca'

--- check: tensorboard_binary_path
INFO: which tensorboard: b'/Users/suman/.local/share/virtualenvs/mlencrypt-research-PwGoqeJm/bin/tensorboard\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 1024>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 1024>
Loopback infos: [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('0.0.0.0', 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'SumanthRatnasAir'

--- check: stat_tensorboardinfo
INFO: directory: /var/folders/j8/_wc022w91rvctlq5_t8xqcvc0000gn/T/.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=69201919, st_dev=16777220, st_nlink=2, st_uid=501, st_gid=20, st_size=64, st_atime=1576804089, st_mtime=1576811707, st_ctime=1576811707)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['/Users/suman/.local/share/virtualenvs/mlencrypt-research-PwGoqeJm/lib/python3.6/site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==0.8.1
astor==0.8.0
cachetools==3.1.1
certifi==2019.11.28
cffi==1.13.2
chardet==3.0.4
Click==7.0
cryptography==2.8
cycler==0.10.0
gast==0.2.2
google-auth==1.7.2
google-auth-oauthlib==0.4.1
google-pasta==0.1.8
grpcio==1.25.0
h5py==2.10.0
idna==2.8
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
kiwisolver==1.1.0
Markdown==3.1.1
matplotlib==3.1.2
mlencrypt==0.1
numpy==1.17.4
oauthlib==3.1.0
opt-einsum==3.1.0
pandas==0.25.3
pip==19.3.1
protobuf==3.11.1
pyAesCrypt==0.4.3
pyasn1==0.4.8
pyasn1-modules==0.2.7
pycparser==2.19
pydicom==1.3.0
pyparsing==2.4.5
python-dateutil==2.8.1
pytz==2019.3
requests==2.22.0
requests-oauthlib==1.3.0
rsa==4.0
scipy==1.3.3
seaborn==0.9.0
setuptools==42.0.1
six==1.13.0
tensorboard==2.0.2
tensorflow==2.0.0
tensorflow-estimator==2.0.1
termcolor==1.1.0
urllib3==1.25.7
Werkzeug==0.16.0
wheel==0.33.6
wrapt==1.11.2

``````

</details>

### Next steps

No action items identified.

## Issue description

![Screen Shot 2019-12-19 at 22 26 38](https://user-images.githubusercontent.com/31281983/71227955-c734c680-22ae-11ea-81ed-db629a8f719e.png)

In the screenshot attached, the variable `Time` appears on the x-axis but also on the y-axis, when I only want it on the x-axis. Here's a shortened version of my HParams config:
```python
hparams = [HP_UPDATE_RULE, HP_K, HP_N,
                   HP_L, HP_KEY_LENGTH, HP_IV_LENGTH]
logdir = 'logs/hparams/' + str(datetime.now())
with tf.summary.create_file_writer(logdir).as_default():
    hp.hparams_config(
        hparams=hparams,
        metrics=[hp.Metric('time_taken', display_name='Time'),
                         hp.Metric('eve_score', display_name='Eve sync')]
    )
```
After each run I run `hp.hparams(current_hparams)`. I've hidden some hyperparameters in the screenshot for the sake of conserving space but they shouldn't affect this bug.

**Comment by wchargin** [comment]
Hi @sumanthratna! This is by design. We show metrics on both axes so
that you can observe correlation between different metrics: for
instance, low cross-correlation between training and validation accuracy
could suggest overfitting.

Can you elaborate on why you’d prefer to only see metrics on the x-axis?
All the metrics should come after all the hyperparameters, so it
hopefully shouldn’t be too intrusive on the space density, as you can
just ignore plots “below the fold” if they’re not interesting to you.

**Comment by sumanthratna** [comment]
That's interesting, sorry for labeling this as a bug. That's a valid reason to keep metrics on the x-axis, but I think it would be nice to only show metrics on the x-axis if they aren't on the y. 

For example, in the screenshot attached, since time is set to linear, it is known that the plot for Time vs. Time will follow the graph of y=x. Instead of showing Time and Eve sync on both rows, just show Eve sync on the first row and Time on the second.
